Source/WebCore/rendering/style/StyleRareInheritedData.cpp

```diff
@@ -81,6 +81,12 @@ StyleRareInheritedData::StyleRareInheritedData()
     , textStrokeColor(RenderStyle::initialTextStrokeColor())
     , textFillColor(RenderStyle::initialTextFillColor())
     , textEmphasisColor(RenderStyle::initialTextEmphasisColor())
+    , visitedLinkTextStrokeColor(RenderStyle::initialTextStrokeColor())
+    , visitedLinkTextFillColor(RenderStyle::initialTextFillColor())
+    , visitedLinkTextEmphasisColor(RenderStyle::initialTextEmphasisColor())
+    , caretColor(RenderStyle::currentColor())
+    , visitedLinkCaretColor(RenderStyle::currentColor())
+    , accentColor(RenderStyle::currentColor())
```

Comment:
This is wrong, and we should fix it in a different bug, since the initial value of `accent-color` is `auto`, not `currentColor`, but it doesn't matter too much since `hasAutoAccentColor` takes care of it anyway.

Anyway, not blocking on this :)

Reply:
Yep this is exactly the reason why I haven't create a specific named initializer for them (such as `initialAccentColor()`).
We should find a proper way to deal with `auto`.